console/src/components/Services/Data/Schema/Schema.js

```diff
@@ -317,6 +326,16 @@ class Schema extends Component {
             dispatch(addExistingTableSql());
           };
 
+          const schemeWarning = !table.schemeWarning ? (
```

Comment:
rename to `gqlCompatibilityWarning`. Move the logic to a common function called `getGqlCompatibilityWarning`. Same logic is repeated in `DataSubSidebar.js`